orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -16,6 +16,11 @@
 import warnings
 from torch.utils.dlpack import from_dlpack, to_dlpack
 
+def torch_tensor_to_dlpack(tensor):
+    if tensor.device.type == 'ort':
+        return C.ort_to_dlpack(tensor)
+    else:
+        return to_dlpack(tensor)
```

Comment:
Cgheck whether https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/ortmodule/_utils.py has this utility already. If not, move it to that file

Reply:
moved to _utils.py